packages/esm-appointments-app/src/home-appointments/common.ts

```diff
@@ -14,14 +9,11 @@ export const launchCheckInAppointmentModal = (appointmentUuid: string) => {
 };
 
 export const handleUndoAction = async (appointmentUuid, mutate) => {
-  onClickUndo();
   const abortController = new AbortController();
   const { status } = await undoAppointmentStatus(appointmentUuid, abortController);
   if (status === 200) {
-    closeActionableNotification();
     mutate(`/ws/rest/v1/appointment/appointment/all`);
   } else {
-    closeActionableNotification();
     showNotification({
       title: 'Error Undoing',
```

Comment:
Is this a valid text?

Reply:
Yes there is also the `InlineNotofication` and that's what we use for errors.